Bots/JavaScript/Consumers/CodeFirst/SimpleHostBot/.env

```
@@ -2,6 +2,20 @@ MicrosoftAppId=
 MicrosoftAppPassword=
 SkillHostEndpoint=http://localhost:3978/api/skills/
```

```suggestion
SkillHostEndpoint=http://localhost:36000/api/skills/
```

Also update the code so the bot starts on this port